doc/api/http.md

```diff
@@ -822,6 +822,10 @@ changes:
     description: The `rawPacket` is the current buffer that just parsed. Adding
                  this buffer to the error object of `'clientError'` event is to
                  make it possible that developers can log the broken packet.
+  - version: v12.0.0
```

Comment:
This should be `REPLACEME` instead of a specific version.

Reply:
thanks will fix